### PR TITLE
Basic support for JM60 keyboard

### PR DESCRIPTION
**Pull request by mkninc**
This adds basic support for JM60 RGB PCB that was sold
by KBDfans https://kbdfans.myshopify.com/
https://www.aliexpress.com/store/product/JM60-RGB60-Custom-mechanical-keyboard-PCB-ANSI/2230037_32810956675.html

No support for RGB Backlight yet.

### CODE REVIEW

File: keyboards/jm60/config.h [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#define PREVENT_STUCK_MODIFIERS
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6464
+#define DEVICE_VER      0x0001
+/* in python2: list(u"whatever".encode('utf-16-le')) */
+/*   at most 32 characters or the ugly hack in usb_main.c borks */
+#define MANUFACTURER "QMK"
+#define USBSTR_MANUFACTURER    'T', '\x00', 'M', '\x00', 'K', '\x00', ' ', '\x00', '\xc6', '\x00'
```
Comment (skullydazed) [inline keyboards/jm60/config.h]: Could you update the manufacturer string please? It should not say TMK or QMK.

File: keyboards/jm60/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+   * |-----------------------------------------------------------|
+   * |CAPS   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Return |
+   * |-----------------------------------------------------------|
+   * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift     |
+   * |-----------------------------------------------------------|
+   * |Ctrl|Gui |Alt |      Space            |Alt |Gui |FN  |Ctrl |
+   * `-----------------------------------------------------------'
+   */
+[_BL] = KEYMAP_ANSI(
+  F(0),    KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_BSPC, \
```
Comment (skullydazed) [inline keyboards/jm60/keymaps/default/keymap.c]: Could you replace `F(0)` with `KC_GESC`? That would allow you to remove everything after line 46.

**Comment by skullydazed** [comment]
Thanks for submitting this! Overall it looks great and once the other two issues have been addressed we can accept this.

**Comment by jackhumbert** [comment]
Awesome :) thanks!

**Pull request merged**